pkg/operator/starter.go

```diff
@@ -310,6 +332,29 @@ func withCustomTags(infraLister v1.InfrastructureLister) deploymentcontroller.De
 			}
 			container.Args = append(container.Args, tagsArgument)
 		}
+		recorder.Eventf("Successfully updated the user tags", "Tag Pairs: %s", tagPairs)
```

Comment:
This function will be called with every `ClusterCSIDriver` CR update, which can be quite often during cluster installation. It should emit an event if anything really changes, not on every successful sync.

Reply:
Agreed.